src/test/java/eu/dissco/exportjob/service/DwcDpServiceTest.java

```diff
@@ -556,4 +477,142 @@ public static JsonNode givenSpecimenJsonOther() throws JsonProcessingException {
+    given(databaseRepository.getRecords(anyString(), eq(0), eq(10))).willReturn(
+        List.of());
+    var dbResponse = new ArrayList<byte[]>();
+    for (int i = 0; i < 18; i++) {
+      dbResponse.add(HexFormat.of().parseHex(
+          "ACED00057372002E65752E64697373636F2E6578706F72746A6F622E646F6D61696E2E64776364702E44774344704D6174657269616C000000000000000102002F4C00136173736F63696174656453657175656E6365737400124C6A6176612F6C616E672F537472696E673B4C000D636174616C6F674E756D62657271007E00014C000B636F6C6C6563746564427971007E00014C000D636F6C6C65637465644279494471007E00014C000E636F6C6C656374696F6E436F646571007E00014C000C636F6C6C656374696F6E494471007E00014C00136461746147656E6572616C697A6174696F6E7371007E00014C000E646174654964656E74696669656471007E00014C001164657269766174696F6E4576656E74494471007E00014C000E64657269766174696F6E5479706571007E00014C001B6465726976656446726F6D4D6174657269616C456E74697479494471007E00014C000B646973706F736974696F6E71007E00014C00076576656E74494471007E00014C001765766964656E6365466F724F6363757272656E6365494471007E00014C0018686967686572436C617373696669636174696F6E4E616D6571007E00014C0018686967686572436C617373696669636174696F6E52616E6B71007E00014C00186964656E74696669636174696F6E5265666572656E63657371007E00014C00156964656E74696669636174696F6E52656D61726B7371007E00014C00206964656E74696669636174696F6E566572696669636174696F6E53746174757371007E00014C000C6964656E746966696564427971007E00014C000E6964656E7469666965644279494471007E00014C0013696E666F726D6174696F6E5769746868656C6471007E00014C000F696E737469747574696F6E436F646571007E00014C000D696E737469747574696F6E494471007E00014C00186973506172744F664D6174657269616C456E74697479494471007E00014C00106D6174657269616C43617465676F727971007E00014C00106D6174657269616C456E74697479494471007E00014C00156D6174657269616C456E7469747952656D61726B7371007E00014C00126D6174657269616C456E746974795479706571007E00014C00126D6174657269616C5265666572656E63657371007E00014C000E6F626A6563745175616E7469747971007E00014C00126F626A6563745175616E746974795479706571007E00014C00136F74686572436174616C6F674E756D6265727371007E00014C00146F776E6572496E737469747574696F6E436F646571007E00014C00126F776E6572496E737469747574696F6E494471007E00014C000C7072657061726174696F6E7371007E00014C000C7265636F72644E756D62657271007E00014C000E736369656E74696669634E616D6571007E00014C000C7461786F6E466F726D756C6171007E00014C00077461786F6E494471007E00014C00097461786F6E52616E6B71007E00014C000C7461786F6E52656D61726B7371007E00014C00137479706544657369676E6174696F6E5479706571007E00014C000A7479706553746174757371007E00014C000C74797069666965644E616D6571007E00014C0016766572626174696D4964656E74696669636174696F6E71007E00014C000D766572626174696D4C6162656C71007E000178707070707070707070707070707400093534343339383634307070707070707070707074001968747470733A2F2F726F722E6F72672F303536366266623936707074002068747470733A2F2F646F692E6F72672F544553542F5A5A5A2D5938562D304234707070707070707074001957686F6C654F7267616E69736D2028616972206472696564297070707070707070707070"));
```

Comment:
is this the same string as below? maybe save this separately as a string so you don't have this massive string in the tests?